### ISSUE TITLE
I can't login to Notion.

### ISSUE THREAD

**Issue by robynrbieber** [Ellpeck/ObsidianCustomFrames#34]
Thank you for making this plugin!
I can't seem to login into Notion. What settings do I have to tick to login?

**Comment by Ellpeck** [comment]
Hi! Depends what versions of the plugin and Obsidian you're on, as well as what issue occurs when you try to log in. 

If you're trying to use this plugin on mobile, websites with logins generally don't work. See [this part of the README](https://github.com/Ellpeck/ObsidianCustomFrames#-on-obsidian-mobile) for more info on that.

**Comment by robynrbieber** [comment]
I amusing the latest version which is v0.14.12.

**Comment by robynrbieber** [comment]
I am using v0.14.12 which is the latest version.

**Comment by Gourav1910** [comment]
> I am using v0.14.12 which is the latest version.

Hi, I was also facing this issue but logged in to notion at the end. I exactly don't remember how I did it because I used the trial and error method. But I recommend you update obsidian and also log in to obsidian from here.

![image](https://user-images.githubusercontent.com/101202140/174152853-a838fa1b-4897-43b2-ade5-de025671ab96.png)


And add some other custom frame of some other app for example google calendar or any other and try to login in there from the same account that you gonna use in notion(I used a google account). I am not sure whether it will work or not but these are the things I tried and somehow I logged in. So try various things, it is possible.
